src/resources/theme.js

```diff
@@ -11,6 +11,16 @@ export default {
       }
     })
   },
+  getAllVersions() {
+    return Fliplet.Env.get('development') ? Promise.resolve() : Fliplet.API.request({
+      url: [
```

Comment:
why don't you search by package? I think we're only looking for one specific theme, correct?

Reply:
`package=com.fliplet.theme.default`